src/main/resources/react4xp/_entries/Dashboard/components/RefreshStatisticsForm.jsx

```diff
@@ -9,7 +13,14 @@ export function RefreshStatisticsForm(props) {
   } = props
 
   const [owners, setOwners] = useState([])
-  const [fetchPublished, setFetchedPublished] = useState(null)
+  const [validated, setValidated] = useState(false)
```

Comment:
Lint warning: 'setValidated' is assigned a value but never used  no-unused-vars

Reply:
Was so sure I had fixed that, but no. Thanks!